bin/kano-settings-onboot

```diff
@@ -30,35 +58,28 @@ from kano_settings.system.audio import is_HDMI, set_to_HDMI
 from kano_settings.system.overclock_chip_support import check_clock_config_matches_chip
 from kano.utils.hardware import get_rpi_model, get_board_property
 
-logger.force_log_level('info')
-
-dry_run = '--dry-run' in sys.argv
-if dry_run:
-    print "dry_run_mode"
-    set_dry_run()
-    logger.force_log_level('debug')
-    logger.force_debug_level('debug')
-    
-
-screen_log_path = '/boot/screen.log'
-
-overrides = {
-    '32V3H-H6A': {'target_group': 'DMT', 'target_mode': 16, 'is_monitor': True},
-    'AS4637_______': {'target_group': 'DMT', 'target_mode': 16, 'is_monitor': True},
-    'BMD_HDMI': {'target_group': 'CEA', 'target_mode': 33, 'is_monitor': True},
-}
-
-
-
 
 def check_model_present(model):
+    '''
+    Returns True if the requested screen model is active
+    '''
     if get_config_comment('kano_screen_used', model):
-        logger.info('The unit was configured for this model already, exiting.')
-        sys.exit()
+        return True
+    else:
+        return False
 
 
 def override_models():
-    global overrides, edid, model
+    '''
+    This function sets specific modes for screens that require special treatment
+    '''
+    overrides = {
+        '32V3H-H6A':     { 'target_group': 'DMT', 'target_mode': 16, 'is_monitor': True },
+        'AS4637_______': { 'target_group': 'DMT', 'target_mode': 16, 'is_monitor': True },
+        'BMD_HDMI':      { 'target_group': 'CEA', 'target_mode': 33, 'is_monitor': True },
```

Comment:
Maybe add a comment to each of these screens to describe what they are (first 2 I think are the screen kit)